src/github/index.ts

```diff
@@ -10,12 +10,15 @@ import { getLogger } from '../config/logger'
 import { metricWebhooks } from '../config/metric-names';
 
 export default (robot) => {
-  const logger = getLogger('github');
+  const logger = getLogger('github.webhooks');
 
   // TODO: Need ability to remove these listeners, especially for testing...
   robot.on('*', (context) => {
     const { name, payload } = context;
-    logger.info({ event: name, action: payload.action }, 'Event received');
+
+    context.log = logger.child({id: context.id})
```

Comment:
Later we should add a proper tracing in our app. We need to put this ID in our queue payloads, and use it to trace our logs. Same for Installation jobs, we need to have a trace ID there, so we can query all logs related to the same sync. But this is quite a big task and out of scope of this PR